packages/xterm.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Xterm < Package
+  description 'The xterm program is a terminal emulator for the X Window System.'
+  homepage 'https://invisible-island.net/xterm/'
+  version '333'
+  source_url 'ftp://ftp.invisible-island.net/xterm/xterm-333.tgz'
```

Comment:
`source_url 'https://invisible-mirror.net/archives/xterm/xterm-333.tgz'` ?

Reply:
ftp should die a slow and painful death.

Reply:
Along with versioned packages?

Reply:
What do you mean?  This is a new package, right?

Reply:
OK. Got it.

Reply:
Curious to know, what is the time difference between:
`wget https://invisible-mirror.net/archives/xterm/xterm-333.tgz`
versus
`wget ftp://ftp.invisible-island.net/xterm/xterm-333.tgz` ?

Reply:
Done.